.github/workflows/ci.yml

```diff
@@ -6,14 +6,20 @@ on:
   pull_request: ~
 
 jobs:
-  # derived from https://dev.to/felipetofoli/github-actions-for-net-full-framework-build-and-test-299h
+  Style:
```

Comment:
Let's call this `lint` so it matches our other library CI workflows

Reply:
Resolved in 36f7579